run_gemini.py

```diff
@@ -0,0 +1,64 @@
+import google.generativeai as genai
+import os
+import subprocess
+import logging
+from dotenv import load_dotenv
+
+
+def get_review(api_key, commit_branch, main_branch, logger):
+    subprocess.run(
+        f"git fetch origin {main_branch} && git fetch origin {commit_branch} && git checkout {commit_branch}",
+        shell=True,
+        check=True,
+    )
+    git_diff = subprocess.run(
+        f"git diff origin/{main_branch}..origin/{commit_branch}",
+        shell=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        check=True,
+    )
+    if git_diff.stderr != b"":
+        logger.error(f"stderr:{git_diff.stderr}")
+
+    genai.configure(api_key=api_key)
+    safety_settings = [
+        {"category": "HARM_CATEGORY_HARASSMENT", "threshold": "BLOCK_NONE"},
+        {"category": "HARM_CATEGORY_HATE_SPEECH", "threshold": "BLOCK_NONE"},
```

Comment:
In `run_gemini.py`, the code generates the jsonl format by manually escaping the characters.  The script can use the `json.dumps()` function to generate the expected jsonl format instead of doing it manually. This could also lead to problems in the future if the format is updated.